### PR TITLE
Feature 1

### PR DESCRIPTION
**Pull request by Namrata-J**
#1 here i have added ui for different pages, i.e. landing, home, labels, archive, trash, profile.
accordingly required, different components with basic ui has been added!!

Have not made them responsive yet!! Will work on that part later!!

please review it, thankyou!!

image
image

### CODE REVIEW

File: src/components/Header/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+            <div className="tn_header-content">
+                <Link to="/" className="tn_header-heading"><h4 className="tn_header-heading">TakeNotes</h4></Link>
+                <input type="text" placeholder="Search" className="tn_header-searchBar b-rad1" />
+                <div className="tn_icons-section">
+                    <GiToggles className="tn_header-settings-icon header-icon" onClick={() => getThePopUpFilterAndSortDisplay()} />
+                    <AiOutlineSetting className="tn_header-settings-icon header-icon" />
+                    <BsSunFill className="tn_header-lightmode-icon header-icon" />
+                    <Link to="profile">
+                        <img className="tn_header-profile-icon et_circular-avatar xs-avatar b-rad4" src="https://cdn.pixabay.com/photo/2015/12/03/08/50/paper-1074131__340.jpg" alt="profile" />
+                    </Link>
```
Comment (JantuDeb) [inline src/components/Header/Header.jsx:32]: `to="/profile`

**Review by JantuDeb — APPROVED** [review]
The UI is really awesome. Pointed out an issue otherwise it is fine

File: src/components/Header/Header.jsx (hunk 1 above)
Comment (Namrata-J) [reply src/components/Header/Header.jsx:32]: yeah!! thanks for pointing that out, didn't notice that mistake!!

File: src/components/SideBar/SideBar.jsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import "./sideBar.css";
+import { AiOutlineHome } from "react-icons/ai";
+import { MdLabelOutline } from "react-icons/md";
+import { BsArchive, BsTrash } from "react-icons/bs";
+import { CgProfile } from "react-icons/cg";
+import { Link } from "react-router-dom";
+
+const sideBarLinksArr = [
```
Comment (sushil-mod) [inline src/components/SideBar/SideBar.jsx:8]: I liked how you used array to render sidebar items

**Review by sushil-mod — APPROVED** [review]
Ui looking nice .nice work on css

**Pull request merged**